deps.sh

```diff
@@ -555,28 +639,21 @@ if [[ $INSTALL_OPENCV == true ]] && [[ ! -f $BUILD_DIR/opencv.done ]]; then
               -D BUILD_opencv_xfeatures2d=OFF \
               -D WITH_PROTOBUF=OFF \
               -D BUILD_PROTOBUF=OFF \
-              -D BUILD_opencv_dnn=OFF \
               -D OPENCV_EXTRA_MODULES_PATH=$BUILD_DIR/opencv_contrib/modules \
-              $(echo $CMDS) -DCMAKE_PREFIX_PATH=$(echo $PY_EXTRA_CMDS) \
-              .. && \
+              -D WITH_INF_ENGINE=ON \
```

Comment:
What does this flag do?

Reply:
It allows OpenCV to use OpenVINO's Inference Engine if it is available.